featuretools/tests/dfs_tests/test_deep_feature_synthesis.py

```diff
@@ -264,6 +264,38 @@ def test_makes_agg_features_with_where(es):
                               'COUNT(log WHERE products.department = food)'))
 
 
+def test_make_groupby_features(es):
+    dfs_obj = DeepFeatureSynthesis(target_entity_id='log',
+                                   entityset=es,
+                                   agg_primitives=[],
+                                   trans_primitives=[],
+                                   groupby_transform_primitives=['cum_sum'])
+    features = dfs_obj.build_features()
+    assert (feature_with_name(features,
+                              "CUM_SUM(value) by session_id"))
+
+
+def test_make_groupby_features_with_agg(es):
```

Comment:
what does this this test test that isn't covered above? is the idea to show that agg primitives get stacked on top of groupby transform features?